### PR TITLE
Settings: Update Ukrainian translation

### PR DESCRIPTION
**Pull request by LetrixX**

### CODE REVIEW

**Comment by LetrixX** [comment]
battery_health string to update slash

File: res/values-uk/yaap_strings.xml [hunk 1]
```diff
@@ -219,8 +232,130 @@
...
+    <string name="doze_gesture_ambient_summary">Показувати Ambient Display замість повного виведення екрана з режиму сну</string>
+    <string name="gesture_wake_ambient">Показувати Ambient</string>
+    <string name="gesture_wake">Повне прокидання</string>
+
+    <!-- wake gestures on AOD -->
+    <string name="doze_gesture_allow_ambient_title">Дозволити на Ambient</string>
+    <string name="doze_gesture_allow_ambient_summary">Дозвольте цьому жесту завжди працювати і на Ambient Display</string>
+
+    <!-- wake gestures haptics -->
+    <string name="doze_gesture_vibrate_title">Тактильний зворотний зв'язок</string>
```
Comment (idoybh) [inline res/values-uk/yaap_strings.xml]:
```suggestion
    <string name="doze_gesture_vibrate_title">Тактильний зворотний зв\'язок</string>
```

File: res/values-uk/yaap_strings.xml [hunk 2]
```diff
@@ -219,8 +232,130 @@
...
+
+    <!-- extra dim schedule -->
+    <string name="extra_dim_schedule_title">Розклад</string>
+
+    <!-- Screenshot shutter -->
+    <string name="screenshot_shutter_sound_title">Звук скриншота</string>
+
+    <!-- APM Enhancement -->
+    <string name="apm_enhancement_title">Покращення режиму польоту</string>
+    <string name="apm_enhancement_summary_on">Запам'ятати стан Bluetooth і WiFi</string>
```
Comment (idoybh) [inline res/values-uk/yaap_strings.xml]:
```suggestion
    <string name="apm_enhancement_summary_on">Запам\'ятати стан Bluetooth і WiFi</string>
```

File: res/values-uk/yaap_strings.xml [hunk 3]
```diff
@@ -219,8 +232,130 @@
...
+    <string name="max_music_volume_title">Медіа</string>
+    <string name="max_call_volume_title">Дзвінки</string>
+    <string name="max_alarm_volume_title">Будильник</string>
+    <string name="steps_unit">кроки</string>
+
+    <!-- Battery Info: Technology -->
+    <string name="battery_technology">Технологія</string>
+	
+    <!-- Battery Info: Health -->
+    <string name="battery_health">Здоров/'я</string>
```
Comment (idoybh) [inline res/values-uk/yaap_strings.xml]:
```suggestion
    <string name="battery_health">Здоров/\'я</string>
```

**Review by idoybh — CHANGES_REQUESTED** [review]

**Review by idoybh — APPROVED** [review]
Now it's good. Thanks!

**Pull request merged**